omero/sysadmins/server-permissions.rst

```diff
@@ -307,11 +311,11 @@ Key
         objects such as images.
 
     Move between groups
-        Only the admin has the right to move other users’
+        Only an admin has the right to move other users’
         data between groups.
 
         .. note::
-            The admin does not have to be member of the
+            An admin does not have to be member of the
```

Comment:
An admin does not have to be member of neither the original, nor the destination group.